### DISCUSSION TITLE
Emty Boot List

### DISCUSSION THREAD

**Discussion by Sinuhe96** [Neverous/efibooteditor#235] [Q&A]
When I start the program on Windows 11, the boot list is empty, even though Windows is booted in secure mode and an EFI partition is available. I thought the program should automatically read all boot entries into this list, shouldn't it? How else can I edit the boot entries? There was no warning, error message or anything. I don't know what to do.

![Screenshot 2024-06-06 010810](https://github.com/Neverous/efibooteditor/assets/111000710/deade0c4-8684-4b5c-bc9c-077c60f5119e)

**Comment by Neverous** [comment]
Probably related to https://github.com/Neverous/efibooteditor/issues/80

You can try `bcdedit /enum firmware` in Terminal as Administrator to see what it shows. [EasyUEFI](https://www.easyuefi.com/index-us.html) was also mentioned to work. 

So far I have no idea what the issue might be in EFI Boot Editor as it works fine everywhere I can try :disappointed:
